catalyst_voices/packages/catalyst_voices_shared/lib/src/platform/io_platform.dart

```diff
@@ -0,0 +1,26 @@
+import 'dart:io';
+
+final class CatalystPlatform {
+  static bool get isAndroid => Platform.isAndroid;
+
+  static bool get isDesktop =>
+      Platform.isLinux || Platform.isMacOS || Platform.isWindows;
+
+  static bool get isFuchsia => Platform.isFuchsia;
+
+  static bool get isIOS => Platform.isIOS;
+
+  static bool get isLinux => Platform.isLinux;
+
+  static bool get isMacOS => Platform.isMacOS;
+
+  static bool get isMobile => Platform.isAndroid || Platform.isIOS;
+
+  static bool get isMobileWeb => false;
```

Comment:
Am correct that it means that `isMobileWeb`, `isWeb` etc. does not supported right now ?

Reply:
Yes, io doesn't work on the web and never will.